src/resources/svg/ForgotPINInstructions.js

```diff
@@ -0,0 +1,714 @@
+import * as React from 'react'
+import Svg, {
+  Defs,
+  LinearGradient,
+  Stop,
+  G,
+  Text,
+  TSpan,
+  Path,
+  Rect,
+  Image,
+} from 'react-native-svg'
+
+function ForgotPIN(props) {
+  return (
+    <Svg width={324} height={469} viewBox="0 0 324 469" {...props}>
+      <Defs>
+        <LinearGradient
+          x1="50%"
+          y1="4.63%"
+          x2="50%"
+          y2="59.739%"
+          id="prefix__a">
+          <Stop stopOpacity={0} offset="0%" />
+          <Stop stopOpacity={0.965} offset="50.09%" />
+          <Stop offset="100%" />
+        </LinearGradient>
+      </Defs>
+      <G fill="none" fillRule="evenodd">
+        <G transform="translate(16)">
+          <Text
+            transform="translate(141)"
+            fill="#5251C1"
+            fillRule="nonzero"
+            fontFamily="TTCommons-Regular, TT Commons"
+            fontSize={14}>
+            <TSpan x={12.52} y={10}>
+              {'Repeat your phrase '}
+            </TSpan>
+            <TSpan x={28.865} y={24}>
+              {'word by word'}
+            </TSpan>
+          </Text>
+          <G transform="translate(48 64)">
+            <Text
+              transform="translate(80 18)"
+              fill="#FFF"
+              fillOpacity={0.9}
+              fillRule="nonzero"
+              fontFamily="TTCommons-Regular, TT Commons"
+              fontSize={19.04}>
+              <TSpan x={0.48} y={13.869}>
+                {'future'}
```

Comment:
Same

Reply:
Regarding the suggestions, I don't think this is very important. Suggestions are in english anyways